pyfr/denserkschemes/__init__.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+from pkg_resources import resource_listdir, resource_string
+import re
+
+import numpy as np
+
+
+class BaseTabulatedDensePseudoStepper(object):
+    def __init__(self, rule):
+        a = []
+
+        rule = re.sub(r'(?<=\))\s*,?\s*(?!$)', r'\n', rule)
```

Comment:
Unsure how much of this is needed.  (We have it for quadrature rules as you can specify them in the ini file.)